(Doc+) link videos for allocation and ilm (#116880)
elastic/elasticsearch: docs/reference/troubleshooting/common-issues/diagnose-unassigned-shards.asciidoc

include::{es-ref-dir}/tab-widgets/troubleshooting/data/diagnose-unassigned-shards-widget.asciidoc[]

> See this https://www.youtube.com/watch?v=v2mbeSd1vTQ[monitoring Allocation health video]
> for an example troubleshooting walkthrough.

Reviewer: ```suggestion
See https://www.youtube.com/watch?v=v2mbeSd1vTQ[this video]
for a walkthrough of monitoring allocation health.
```
